pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "mikrotik-exporter";
+  version = "20200210-${stdenv.lib.strings.substring 0 7 src.rev}";
```

Thanks for the adding the exporter with a module and test :+1:

Just one minor nitpick, could you change the version format to the following combination of "unstable-" and the ISO-8601 date for the revision?
```suggestion
  version = "unstable-2020-02-10";
```

---

The `-unstable-` infix should be part of `(p)name` because repology: #68518

---

Updated, please take a look.

---

@Ma27 oh thanks for noticing, I wasn't aware of that issue :+1: